plugins/prefix/plugin_test.go

```diff
@@ -80,3 +80,14 @@ func TestRoundTrip(t *testing.T) {
 			iapd.Options.Prefixes())
 	}
 }
+
+func TestDup(t *testing.T) {
+	_, prefix, err := net.ParseCIDR("2001:db8::/48")
+	if err != nil {
+		panic("bad cidr")
```

Comment:
shouldn't we use `t.Fatalf` or `t.Panicf` here? And more in general we should consolidate everything to use `stretchr/testify`

Reply:
not really, this isn't the test failing, it's a bug in writing the test (if `net.ParseCIDR("2001:db8::/48")` fails we have issues). I'm using it kind of like rust's `unwrap()`. Maybe there's a better idiom ?

`testify` was only introduced in this repo in the very latest PR. the dhcp lib has been using it for a while but so far barely anything uses it in coredhcp, I'm not opposed to it but I don't think we've actively decided to consolidate on it before this ?

Reply:
Some packages use `Must*` versions of functions like this, e.g. `MustParseCIDR`, which basically wrap a `panic` in case of error. Since there's no such function here, and this is a one-off, I'd just leave `panic`. Feel free to merge when you think its' ready :)